momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -25,20 +27,16 @@ public final class CacheClient implements Closeable {
   private final ScsDataClient scsDataClient;
 
   CacheClient(
-      String authToken,
-      Duration itemDefaultTtl,
-      Optional<OpenTelemetry> telemetryOptional,
-      Optional<Duration> requestTimeout) {
+      @Nonnull String authToken,
+      @Nonnull Duration itemDefaultTtl,
+      @Nullable OpenTelemetry openTelemetry,
```

Comment:
I thought we got rid of this otel stuff?

Reply:
I must have misunderstood you and removed the tests. I can put up a separate pr that removes the rest of it.